### PR TITLE
[REVIEW] Some fixes to dbscan demo notebook in prep for 0.10 release

### PR DESCRIPTION
**Pull request by cjnolet**

### CODE REVIEW

**Comment by taureandyernv** [comment]
@cjnolet did you update based on @JohnZed 's comments?  Waiting on that to merge :)

**Comment by cjnolet** [comment]
Nope. I will get to this tonight or early tomorrow.

**Comment by cjnolet** [comment]
@taureandyernv @JohnZed, implemented the changes. This is ready to go!

File: cuml/dbscan_demo.ipynb [hunk 1]
```diff
@@ -64,17 +66,27 @@
...
+     "output_type": "stream",
+     "text": [
+      "float32\n"
+     ]
+    }
+   ],
    "source": [
     "host_data, host_labels = make_blobs(\n",
-    "   n_samples=n_samples, n_features=n_features, centers=5, random_state=7)\n",
+    "   n_samples=n_samples, n_features=n_features, centers=5, cluster_std=0.1)\n",
```
Comment (JohnZed) [inline cuml/dbscan_demo.ipynb]: Why no more random seed?

File: cuml/dbscan_demo.ipynb [hunk 2]
```diff
@@ -106,9 +120,33 @@
...
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 9,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "KeyboardInterrupt",
+     "evalue": "",
+     "output_type": "error",
```
Comment (JohnZed) [inline cuml/dbscan_demo.ipynb]: This seems to have some output in it. Would be nice to check in clean.

Comment (cjnolet) [reply cuml/dbscan_demo.ipynb]: Good catch! I may not have hit save when I cleared the output in Jupyter. Fixed.

File: cuml/dbscan_demo.ipynb (hunk 1 above)
Comment (cjnolet) [reply cuml/dbscan_demo.ipynb]: This is what I get for being lazy and attempting to do a merge through the github UI. Fixed.

**Comment by cjnolet** [comment]
Aright, how about now?

**Pull request merged**